docs/tutorials/single_screen.rst

```diff
@@ -37,25 +43,20 @@ Start with some standard imports.
     from astropy import constants as const
 
 Set a colormap to use for phases. Preferably import the perceptually uniform
-colormap from the local file ``hue_cycle_cmap``, available for download
-:download:`here <./hue_cycle_cmap.py>`.
-The ``hsv`` colormap is used as fallback.
+colormap `phasecmap` from :py:mod:`screens.visualization`.
+The `hsv` colormap from :py:mod:`matplotlib.cm` is used as fallback.
 
-.. plot::
-    :include-source:
-    :context:
+.. jupyter-execute::
 
     try:
-        from hue_cycle_cmap import cmap as phasecmap
+        from screens.visualization import phasecmap
```

Comment:
Can this import still fail?

Reply:
This way, one can run the downloadable script/notebook without having `screens` installed.

Reply:
OK, that's fine (though perhaps one can insist on having screens in the screens documentation!)